tutorials/klusterai-api/reliability.ipynb

```diff
@@ -47,7 +47,7 @@
     "  platform</a> if you don't have one\n",
     "  - **A kluster.ai API key**: after signing in, go to the <a href=\"https://platform.kluster.ai/apikeys\" \n",
     "  target=\"_blank\">**API Keys**</a> section and create a new key. For detailed instructions, check out the <a \n",
-    "  href=\"/get-started/get-api-key/\" target=\"_blank\">Get an API key</a> guide"
+    "  href=\"/verify/get-api-key/\" target=\"_blank\">Get an API key</a> guide"
```

Comment:
I thought links in tutorials needed to be the entire link? with `docs.kluster.ai`?